packages/theme-docs/src/pages/_.vue

```diff
@@ -52,11 +52,17 @@ export default {
       return error({ statusCode: 404, message: 'Page not found' })
     }
 
-    const [prev, next] = await $content(app.i18n.locale, { deep: true })
-      .only(['title', 'slug', 'to'])
-      .sortBy('position', 'asc')
-      .surround(document.slug, { before: 1, after: 1 })
-      .fetch()
+    let prev
+    let next
+
+    if (!document.hiddenInMenu) {
+      [prev, next] = await $content(app.i18n.locale, { deep: true })
+        .where({ hiddenInMenu: false })
+        .only(['title', 'slug', 'to'])
+        .sortBy('position', 'asc')
+        .surround(document.slug, { before: 1, after: 1 })
+        .fetch()
+    }
```

Comment:
I added a logic like below, so that sidebar content and prev-next links can be consistent.

- In unhidden content page, only unhidden pages are shown as prev-next links.
- In hidden content page, no prev-next links are shown.